pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -68,9 +97,39 @@ func (pl *NodeAffinity) Filter(ctx context.Context, state *framework.CycleState,
 	if pl.addedNodeSelector != nil && !pl.addedNodeSelector.Match(node) {
 		return framework.NewStatus(framework.UnschedulableAndUnresolvable, errReasonEnforced)
 	}
+	/*
 	if !pluginhelper.PodMatchesNodeSelectorAndAffinityTerms(pod, node) {
```

Comment:
There are several other components depending on `pluginhelper.PodMatchesNodeSelectorAndAffinityTerms`. This change duplicates similar logic with the function, is it possible that we refactor helper functions to still share codes with other components?